gnomad_qc/v4/assessment/calculate_per_sample_stats.py

```diff
@@ -479,26 +529,49 @@ def main(args):
                 vep_canonical=args.vep_canonical,
                 vep_mane=args.vep_mane,
                 rare_variants_afs=rare_variants_afs,
+            ).write(filtering_groups_res.path, overwrite=overwrite)
+
+        if args.create_per_sample_counts_ht:
+            logger.info(
+                "Calculating per-sample variant statistics for %s...", data_type
             )
-            create_per_sample_counts_ht(mt, filter_groups_ht).write(
+            if test:
+                logger.warning(
+                    "This test requires that the full filtering groups Table has been "
+                    "created. Please run --create-filter-group-ht without "
+                    "--test-dataset or --test-n-partitions if it doesn't already exist."
+                )
+            filter_groups_ht = get_summary_stats_filtering_groups(data_type).ht()
+            vds_load_func = (
+                get_gnomad_v4_vds
+                if data_type == "exomes"
+                else get_gnomad_v4_genomes_vds
+            )
+            mt = vds_load_func(
+                test=test_dataset,
```

Comment:
Probably can't be solved in this PR, but genomes test VDS has the same number of partitions as the whole VDS, 115376 for (31236097, 80)  variants x samples, this over-partition makes the test run super slow.

Reply:
Yes, as discussed previously, we should only be focusing on the exomes right now, we can handle these finer details when we get to working with the genomes.